binancial/__init__.py

```
@@ -2,6 +2,7 @@
 from .accountant import Accountant
 
 from . import data
+from . import features
```

Ruff (0.8.2)

`.features` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)